Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_045.PostgreSqlConnectionString.sarif

```diff
@@ -296,57 +296,18 @@
                   "startLine": 9,
                   "startColumn": 45,
                   "endLine": 9,
-                  "endColumn": 157,
-                  "charOffset": 914,
-                  "charLength": 112,
-                  "snippet": {
-                    "text": "Port=5555;Password=my_pw;Connection Pruning Interval=doodle;Host=my_host;Username=my_user;Pooling=true;Port=5555"
-                  }
-                }
-              }
-            }
-          ],
-          "fingerprints": {
-            "ValidationFingerprint/v1": "[acct=my_user][host=my_host][pwd=my_pw][port=5555]"
-          }
-        },
-        {
-          "ruleId": "SEC101/045",
```

Comment:
>SEC101 [](start = 21, length = 6)

the missing results in all SARIF files is the outcome of our change! i.e., we were produce multiple results for what was effectively the same test case. :)